Doc/library/dis.rst

```diff
@@ -20,6 +20,10 @@ interpreter.
    between versions of Python.  Use of this module should not be considered to
    work across Python VMs or Python releases.
 
+   .. versionchanged:: 3.6
+      Use fixed 2 bytes per instruction for all instructions.  Instructions
+      of variable length were used before.
```

Comment:
I would phrase this as "Use 2 bytes for each instruction. Previously the number of bytes varied by instruction."